### PR TITLE
Add support for foreign assets in application txn

### PR DESCRIPTION
**Pull request by algorotem**
This PR adds support for the foreignAssets field

### CODE REVIEW

**Comment by algorotem** [comment]
@algonautshant can you please generate the new indexer client based om the spec in this PR (https://github.com/algorand/indexer/pull/165) and add it to this PR?

File: test/applications_integration_test.go [hunk 1]
```diff
@@ -111,7 +111,7 @@ func waitForTransaction(transactionId string) error {
 func iBuildAnApplicationTransaction(
 	operation, approvalProgram, clearProgram string,
 	globalBytes, globalInts, localBytes, localInts int,
-	appArgs, foreignApps, appAccounts string) error {
+	appArgs, foreignApps, foreignAssets, appAccounts string) error {
```
Comment (winder) [inline test/applications_integration_test.go:114]: This is only added to the unit tests at the moment, so it shouldn't be added here.

**Comment by rotemh** [comment]
> `applications_responsemodels.go` also needs to be updated with `ForeignAssets`

Indeed and I did. Just haven't pushed the commit yet.

File: test/applications_unit_test.go [hunk 2]
```diff
@@ -67,11 +67,17 @@ func iBuildAnApplicationTransactionUnit(
 	if appAccounts != "" {
 		accs = strings.Split(appAccounts, ",")
 	}
+
 	fApp, err := splitUint64(foreignApps)
 	if err != nil {
 		return err
 	}
 
+	fAssets, err := splitUint64(foreignApps)
```
Comment (winder) [inline test/applications_unit_test.go:76]: s/foreignApps/foreignAssets

**Pull request merged**